daml-assistant/src/DAML/Assistant/Env.hs

```diff
@@ -139,7 +141,8 @@ getSdk damlPath projectPathM =
 
         sdkPath <- firstJustM id
             [ fmap SdkPath <$> lookupEnv sdkPathEnvVar
-            , pure (defaultSdkPath damlPath <$> sdkVersion)
+            , autoInstall damlPath sdkVersion
```

Comment:
It looks like `autoInstall` does roughly the same check as `sdkPathFallback` at the beginning if `auto-install` is enabled and if it isn’t we use `sdkPathFallback` anyway. Can we move `sdkPathFallback` before `autoInstall` and remove the check from `autoInstall`?

Reply:
Oh, that's true! :+1: